packages/veritone-widgets/test/helpers/utils.js

```diff
@@ -0,0 +1,20 @@
+import _ from 'lodash';
+// Removes all functions in an object so you can test
+// against objects without errors
+export const removeFunctions = (obj) => {
+  if (!_.isObject(obj)) {
+    return obj;
+  }
+  let temp = {}
+  const keys = _.keys(obj)
```

Comment:
FYI we always write named imports from lodash, so that the bundler can optimize them for tree shaking. Obviously this doesn't matter in a test environment but you should do it anyway for consistency.

`import { isObject, isFunction, keys } from 'lodash'`